packages/backend/src/plugins/techInsights.ts

```diff
@@ -41,11 +41,11 @@ export default async function createPlugin({
     discovery,
     factRetrievers: [
       createFactRetrieverRegistration(
-        '* * * * *', // Example cron, every minute
+        '1 * 1 1 *', // Example cron, At minute 1 on day-of-month 1 in January.
```

Comment:
Does it really make sense to have this enabled when it's only once per year? 😁  It feels like some form of GC is required and perhaps only storing the latest or a smaller set of historical data if that's needed.

Reply:
These are currently mostly example items to display how tech-insights can be used. In reality since there are not that many public fact retrievers yet available, a lot of implementations in use will be internally built ones. The assumption would also be that those internally built ones are configured to be run with a cadence which is sane for the implementation itself.

Reply:
@Xantier the cron would be executed on every hour on January the 1st, see <https://crontab.guru/#1_*_1_1_*>
```log
next at 2023-01-01 00:01:00
then at 2023-01-01 01:01:00
then at 2023-01-01 02:01:00
then at 2023-01-01 03:01:00
then at 2023-01-01 04:01:00
```
How about `1 1 1 1 *` ?
Or `1 1 1 * *` = every month?

Reply:
Swapped to `1 1 1 * *` instead.